### ISSUE TITLE
RCE 10.3 palette

### ISSUE THREAD

**Issue by ArneBachmannDLR** [rcenvironment/rce#40]
Just tested the new release.
The new palette looks nice!

I'm confused, however, regarding the purpose of the custom group feature. Why is there a difference between *User Integrated Tools* and custom groups? And how to insert tools into these groups? I'm not able to drag'n'drop components between the trees (which would require several `configuration.json`s to be modified in concert).

The dialog title states *Manage ... Group assignments* and *Reset components to their default groups*.

I assume this addition is just a preparation for a later feature not available yet? I couldn't find details in the documentation (btw - the screenshots still show the old palette).

**Comment by janflink** [comment]
Hi Arne,

the custom palette groups are independent regarding the group configuration of integrated tools. Moving components to custom groups is a local, profile based, configuration and has no effect for other RCE instances.
You can find the documantion within RCE by using the integrated help. (F1 with focus on the palette or Help Content in the menu).

Best regards,
Jan

**Comment by ArneBachmannDLR** [comment]
Yeah, the context help was useful! Moving a group is not a drag-and-drop operation, but requires an action from the context menu. I wasn't expecting that, but was probably easier to implement.

![grafik](https://user-images.githubusercontent.com/54819781/184804826-c42aef8c-9914-4d9a-929a-d2698a476a85.png)

The "Reset Component" action is also available from within the "Manage Custom Groups" dialog - a bit weird to have a component vanish from inside the dialog, but not to have any option to bring it back (select and add or move component inside the dialog into the grouos). This dialog also has no undo option (OK vs. Cancel), only Close.

**Comment by janflink** [comment]
Hello Arne,

Not enabling drag & drop for group assignments were a conscious usability decision. Since drag-and-drop is used to add components to a workflow and the assignment of components to groups is probably not the daily business for RCE users, we wanted to avoid accidental group assignments. It was not about it was easier to implement.
All operations regarding the groups and group assignments can be accessed via the palette's context menu.
In addition, the 'Manage' dialog should give the opportunity to give a quick overview of all custom groups and group assignements. Furthermore, the assignments of the components currently not available (offline) and empty groups are displayed in the dialog. Group assignments of offline components can also be deleted there.

I hope that helps a little.

Best regards.
